pkg/controller/m3admin_client_test.go

```diff
@@ -87,17 +88,18 @@ func TestNewMultiAdminClient(t *testing.T) {
 		m.plClientFn,
 		m.clusterKeyFn,
 		m.clusterURLFn,
-		m.adminClient,
+		m.adminClientFn,
 		m.logger,
 	} {
 		assert.NotNil(t, v)
 	}
 }
 
-func newM3DBCluster(name string) *myspec.M3DBCluster {
+func newM3DBCluster(ns, name string) *myspec.M3DBCluster {
```

Comment:
Is this assuming a 1:1 association of namespace to cluster?

Reply:
I don't think so, but it's just generating a fake metadata for the tests